### PR TITLE
Update README to point to new site

### PR DESCRIPTION
**Pull request by jmertic**
Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,110 +1,7 @@
-# Overview
+[![Netlify Status](https://api.netlify.com/api/v1/badges/a76820ec-1092-49fb-8538-a82900b711b0/deploy-status)](https://app.netlify.com/sites/aswf-tac/deploys)
 
-The role of the Technical Advisory Committee (TAC) of the Academy Software Foundation (ASWF) is to facilitate communication and collaboration among the Technical Projects. The TAC will be responsible for:
+# Academy Software Foundation Techincal Advisory Council (TAC)
```
Comment (jfpanisset) [inline README.md:3]: Typo: "Technical", not "Techincal"

Comment (jmertic) [reply README.md]: The bane of my spelling challenges - thank you :-)

**Comment by jmertic** [comment]
> More generally, the substance of README.md has been moved to index.md, but someone landing on the repo will now just see:
> https://github.com/AcademySoftwareFoundation/tac/tree/jmertic-patch-7
> which is somewhat surprising: typically when you navigate to a GitHub repo, you expect the README.md to present the overview material for the repo. Does the home page for the netlifyed version have to be in index.md, or could it come straight from README.md, like a GitHub Pages jekyll site?

It's a good question; the challenge is now we a generating parts of that page as part of the Netlify build ( you can see all the tags in there ) so it likely won't look right. I could see over time further divergences, so directing people to the display site is likely where we'd want to go here. But I'm open to direction for the community - so let me know if we should go a different route.

> Also, the sections:
> 
> * Meetings and Conversations
> * Conference Call Details
> 
> that were in README.md are not in index.md, is this desired? Or did that information migrate somewhere else?

That moved under meetings/README.md

**Comment by jfpanisset** [comment]
Looks good, just learned that the C in TAC stands for Council and not Committee.

**Comment by jmertic** [comment]
> Looks good, just learned that the C in TAC stands for Council and not Committee.

Yep! https://github.com/AcademySoftwareFoundation/foundation/blob/master/CHARTER.md#7--technical-advisory-council

**Pull request merged**